### ISSUE TITLE
Gyoku.xml with unwrap: true doesn't set attributes

### ISSUE THREAD

**Issue by drakmail** [savonrb/gyoku#56]
For example:

``` ruby
Gyoku.xml({items: {:content! => [item: "abc"], :@amount => "1"}}, unwrap: true)
# => "<items><item>abc</item></items>"
```

But I'm expecting:

``` ruby
Gyoku.xml({items: {:content! => [item: "abc"], :@amount => "1"}}, unwrap: true)
# => "<items amount=\"1\"><item>abc</item></items>"
```

**Comment by tjarratt** [comment]
Thanks for reporting this issue @drakmail. Would you be interested in submitting a pull request for this?

**Comment by drakmail** [comment]
I'll try to find some time for fixing that and submitting pull request, but couldn't guarantee that.

**Comment by tjarratt** [comment]
This should have been addressed in commit  954d002 by the wonderful @jpmoral.
